packages/riceclf/src/riceclf/main.py

```diff
@@ -6,8 +6,8 @@
 
 # Configuration parameters
 CONFIG = {
-    'dataset_path': r'C:\Users\Path\to\partB\Rice_Cammeo_Osmancik.arff',
-    'output_path': r'C:\Users\Path\to\partB\output',
+    'dataset_path': r'packages\riceclf\src\riceclf\Rice_Cammeo_Osmancik.arff',
```

Comment:
```suggestion
    'dataset_path': r'packages/riceclf/src/riceclf/Rice_Cammeo_Osmancik.arff,
```
The `\` directory separator convention is Windows-specific, and doesn't work for macOS. Let's use the `/` separator as it's supported on Windows, macOS, and Linux.